cmd/src/teams_members_add.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
this requires https://github.com/sourcegraph/sourcegraph/pull/47143 to work which I'm intending to work on tomorrow.